src/fe-gtk/servlistgui.c

```diff
@@ -39,6 +39,12 @@
 #define SERVLIST_X_PADDING 4			/* horizontal paddig in the network editor */
 #define SERVLIST_Y_PADDING 0			/* vertical padding in the network editor */
 
+#ifdef USE_OPENSSL
+# define DEFAULT_SERVER "irc.example.com/6697"
```

Comment:
I think I prefer the previous default of not a real domain.

Reply:
I changed it back.